src/vm/docs/mod.rs

```diff
@@ -264,6 +264,46 @@ Note: This function is only available starting with Stacks 2.1.",
 "#,
 };
 
+const PARSE_PRINCIPAL_API: SimpleFunctionAPI = SimpleFunctionAPI {
+    name: None,
+    signature: "(parse-principal property-name principal-address)",
+    description: "A principal value is a concatenation of two things: a single *version byte*
+(indicating the type of account and the type of network that this principal can spend tokens on),
+and a 20-byte *public key hash* (indicating the principal's unique identity).
+`parse-principal` is used to decompose a principal into its component parts.
```

Comment:
This piece of documentation would be a great place for exhaustively be enumerating all the versions.

Reply:
Ah nevermind seeing it in the other function.